packages/react-server/src/ReactFlightServer.js

```diff
@@ -168,52 +168,6 @@ function getStack(error: Error): string {
   }
 }
 
-function initCallComponentFrame(): string {
-  // Extract the stack frame of the callComponentInDEV function.
-  const error = callComponentInDEV(Error, 'react-stack-top-frame', {}, null);
-  const stack = getStack(error);
-  const startIdx = stack.startsWith('Error: react-stack-top-frame\n') ? 29 : 0;
-  const endIdx = stack.indexOf('\n', startIdx);
-  if (endIdx === -1) {
-    return stack.slice(startIdx);
-  }
-  return stack.slice(startIdx, endIdx);
-}
-
-function initCallIteratorFrame(): string {
-  // Extract the stack frame of the callIteratorInDEV function.
-  try {
-    (callIteratorInDEV: any)({next: null});
-    return '';
-  } catch (error) {
-    const stack = getStack(error);
-    const startIdx = stack.startsWith('TypeError: ')
-      ? stack.indexOf('\n') + 1
-      : 0;
-    const endIdx = stack.indexOf('\n', startIdx);
-    if (endIdx === -1) {
-      return stack.slice(startIdx);
-    }
-    return stack.slice(startIdx, endIdx);
-  }
-}
-
-function initCallLazyInitFrame(): string {
-  // Extract the stack frame of the callLazyInitInDEV function.
-  const error = callLazyInitInDEV({
-    $$typeof: REACT_LAZY_TYPE,
-    _init: Error,
-    _payload: 'react-stack-top-frame',
-  });
-  const stack = getStack(error);
-  const startIdx = stack.startsWith('Error: react-stack-top-frame\n') ? 29 : 0;
-  const endIdx = stack.indexOf('\n', startIdx);
-  if (endIdx === -1) {
-    return stack.slice(startIdx);
-  }
-  return stack.slice(startIdx, endIdx);
-}
-
 function filterDebugStack(error: Error): string {
```

Comment:
This is different from the one in FlightOwnerStack which is for formatting the string for `captureOwnerStack()` for presentation.

This one is for formatting the string into the serializable `stack` field for serialization. This will likely turn into a structured form instead of a string in a follow up.

This could be coming from the current server, however, this Error could also be one that includes fake stack frames it originated on another server. In this case we'd need to do something to reverse engineer it. E.g. stripping the `rsc://React/` prefixes.